lib/wallet/http.js

```diff
@@ -1153,6 +1502,77 @@ class HTTPOptions {
   }
 }
 
+class TransactionOptions {
+  /**
+   * TransactionOptions
+   * @alias module:http.TransactionOptions
+   * @constructor
+   * @param {Validator} valid
+   */
+
+  constructor(valid) {
+    if (valid)
+      return this.fromValidator(valid);
+  }
+
+  /**
+   * Inject properties from Validator.
+   * @private
+   * @param {Validator} valid
+   * @returns {TransactionOptions}
+   */
+
+  fromValidator(valid) {
+    assert(valid);
```

Comment:
I tried this approach, its not actually a `Validator`, its a `MultiValidator` which is not exposed from the `bval` library. Since the `Validator` is exported directly (not as a property of an object),
it makes it more difficult to upgrade. One solution is to add `MultiValidator` as a property of the `Validator` so that its possible to `assert(valid instanceof Validator.MultiValidator)`

https://github.com/bcoin-org/bval/blob/master/lib/validator.js#L836

Reply:
@tynes this is good for now.